src/flux/app-config.coffee

```diff
@@ -4,11 +4,16 @@ flux = require 'flux-react'
 
 {makeSimpleStore} = require './helpers'
 
+MinimizedPattern = /tutor\.min-\d+/
```

Comment:
They're all in `/dist/*.svg` as part of the build so no need for the paths and no need for `.replace("dist", "resources/images")`